### PR TITLE
Documentation website and CI

### PR DESCRIPTION
**Pull request by smehta91**

### CODE REVIEW

File: docs/website/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Website
```
Comment (smehta91) [inline docs/website/README.md]: Remove

File: docs/website/sidebars.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  // But you can create a sidebar manually
+  /*
+  tutorialSidebar: [
+    {
+      type: 'category',
+      label: 'Tutorial',
+      items: ['hello'],
+    },
+  ],
+   */
```
Comment (smehta91) [inline docs/website/sidebars.js]: Remove commented code

File: docs/website/src/pages/markdown-page.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (smehta91) [inline docs/website/src/pages/markdown-page.md]: Remove this File

**Review by smehta91 — COMMENTED** [review]
Remove yarn.lock files

**Review by tusharmath — APPROVED** [review]

**Review by amitksingh1490 — APPROVED** [review]

**Review by amitksingh1490 — APPROVED** [review]

**Review by amitksingh1490 — APPROVED** [review]

**Pull request merged**